references.bib

```diff
@@ -1,3 +1,10 @@
+@Manual{CCDL2020,
+  title = {scRNA-seq Dimension Reduction},
+  author = {CCDL Team},
```

Comment:
One more comment
```suggestion
  author = {CCDL},
```
It thinks CCDL is the first name, Team is last name, so just delete the `Team` part.